be/src/exec/pipeline/set/union_const_source_operator.cpp

```diff
@@ -44,6 +39,20 @@ StatusOr<vectorized::ChunkPtr> UnionConstSourceOperator::pull_chunk(starrocks::R
     return std::move(chunk);
 }
 
+void UnionConstSourceOperator::_move_column(ColumnPtr& dst_column, ColumnPtr& src_column) {
```

Comment:
can  _move_column be replaced by already-existing ColumnHelper::unfold_const_column?

if cannot. add another util function to column_helper.cpp.

when we handle a column, suggest to category column into four classes:
1. const null:  src_column->only_null()
2. const but not null:  src_column->is_constant()
3. nullable: src_column->is_nullable
4. not nullable: else